### PR TITLE
add: coc handling

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
Add Code of Conduct, as per #1.

### CODE REVIEW

**Comment by zeeshanlakhani** [comment]
Nice thing about the CoC in repo, is GH picks up on this and has some nice UI to link to it too.

**Review by bgins — APPROVED** [review]
Looks good! 🎉 

A few suggested edits in the comments.

File: rust+wasm/README.md [hunk 1]
```diff
@@ -128,9 +133,13 @@ development.
 Run `nix develop` or `direnv allow` to load the `devShell` flake output,
 according to your preference.
 
+### Formatting
```
[the remark this answers is not in the document]
Comment (zeeshanlakhani) [reply rust+wasm/README.md:136]: It's inside the if/else.

[the remark this answers is not in the document]
Comment (zeeshanlakhani) [reply rust+wasm/README.md:136]: @bgins but lmk if i missed it :).

[the remark this answers is not in the document]
Comment (bgins) [reply rust+wasm/README.md:136]: Ah yeah, I missed the conditional! LGTM. 👌

**Pull request merged**